### ISSUE TITLE
Version Support Tracker (v2)

### ISSUE THREAD

**Issue by Informatic** [RootMyTV/RootMyTV.github.io#18]
The purpose of this issue is to track which TV models, and firmware versions, the exploit is working against.

If you are having trouble getting the exploit to work, please contact us on discord and/or open a separate issue.

At a minimum, please specify your webOS version, software version, and TV model - feel free to add any other details you think are relevant.

You can find your "webOS Version" under **Settings > General > About This TV > TV Information > webOS TV Version**

You can find your "Software Version" under **Settings > General > About This TV > Software Version**

**IMPORTANT: Please don't get webOS version and software version mixed up!**

**Comment by Informatic** [comment]
**Status:** working
**TV Model:** 43UJ6307-ZA
**webOS Version:** 3.8.0-61312 (dreadlocks2-dunggir)
**Software Version:** 05.80.50

---

Tip: all required info can be extracted using the following command on running system:
```sh
grep -h -E '"(hardware_id|core_os_release|product_id|webos_manufacturing_version)"' /var/run/nyx/*
```

**Comment by AkdM** [comment]
Status: Working
TV Model: OLED65C15LA
webOS Version: 6.2.0-27 (kisscurl-kavir)
Software Version: 3.20.80

**Comment by vordenken** [comment]
**Status**: Working
**TV Model**: 65UN74007LB
**webOS Version**: 5.3.0-21 (jhericurl-jasper)
**Software Version**: 04.30.50

**Comment by janchrillesen** [comment]
Status: working
    "hardware_id": "HE_DTV_W19P_AFADABAA",
    "product_id": "65SM8200PLA",
    "core_os_release": "4.9.3-5390409",
    "webos_manufacturing_version": "05.10.25",

**Comment by TBSniller** [comment]
Status: working  
`"hardware_id": "HE_DTV_W20H_AFADABAA",
    "product_id": "65NANO867NA",
    "core_os_release": "5.3.0-21",
    "webos_manufacturing_version": "04.30.50",
`
Upgraded directly from RootMyTV v1 exploitable version. And then used v2 (Sorry didn't looked up version before update)  
`/home/root/` was untouched, so .ssh auth-keys were persistent during upgrade

**Comment by alanorth** [comment]
**Status**: working
**TV Model**: 43UM7340PVA
**Software Version**: 4.9.3-5390409
**WebOS Version**: 05.10.25

**Comment by elamperti** [comment]
**Status**: working!
**TV Model**: 43LK5700-SC
**WebOS Version**: 4.4.0-9
**Software Version**: 05.40.09

(v1 didn't work before, v2 went smoothly) Thanks for your work ❤️

**Comment by samcday** [comment]
Working. I didn't dare dream that I'd be able to jailbreak my fancy-ass TV and run RetroArch on it. Thank you, heroes!

```
    "hardware_id": "HE_DTV_W21O_AFABATAA",
    "product_id": "OLED48C17LB",
    "core_os_release": "6.2.0-27",
    "webos_manufacturing_version": "03.20.80",
```

**Comment by OnkelBuTscH** [comment]
Status: Working
    "hardware_id": "HE_DTV_W21U_AFADATAA",
    "product_id": "OLED55A19LA",
    "core_os_release": "6.2.0-33",
    "webos_manufacturing_version": "03.21.10"

**Comment by Juanfragar** [comment]
Status: Working
    "hardware_id": "HE_DTV_W16P_AFADABAA",
    "product_id": "65UH625V-ZA",
    "core_os_release": "3.4.0-5805",
    "webos_manufacturing_version": "05.60.15",


Special thanks to @Informatic

**Comment by DpunktKpunkt** [comment]
Status: Working

    "hardware_id": "HE_DTV_W18A_AFADABAA",
    "product_id": "65UK6400PLF",
    "core_os_release": "4.4.0-9",
    "webos_manufacturing_version": "05.40.09",

**Comment by romland** [comment]
Status: Working
`
    "hardware_id": "HE_DTV_W20O_AFABABAA",
    "product_id": "OLED55CX6LA",
    "core_os_release": "5.3.0-10",
    "webos_manufacturing_version": "04.30.10",
`

and status: working
`
    "hardware_id": "HE_DTV_W20O_AFABABAA",
    "product_id": "OLED55CX6LA",
    "core_os_release": "5.3.0-22",
    "webos_manufacturing_version": "04.30.55",
`

**Comment by ccvh** [comment]
Status: working 
TV Model: 55NANO869PA
webOS Version: 6.2.0-31 kissurl-kavir
Software Version: 03.21.00

**Comment by DpunktKpunkt** [comment]
Status: not working (slide / "5" doesn't work and won't start exploit)
TV Model: 65UJ6309
webOS: ?
Software Version: 6.00.20

**Comment by DpunktKpunkt** [comment]
Status: working
    "hardware_id": "HE_DTV_W17P_AFADABAA",
    "product_id": "65UJ6309-ZA",
    "core_os_release": "3.9.0-62906",
    "webos_manufacturing_version": "06.00.20",

**Comment by solaris007** [comment]
Status: working
```
    "hardware_id": "HE_DTV_W18O_AFABABAA",
    "product_id": "OLED65C8PLA",
    "core_os_release": "4.4.0-9",
    "webos_manufacturing_version": "05.40.09",
```

**Comment by vpont** [comment]
Working!
```
    "hardware_id": "HE_DTV_W20O_AFABABAA",
    "product_id": "OLED65CX6LA",
    "core_os_release": "5.3.0-22",
    "webos_manufacturing_version": "04.30.55",
```

**Comment by th0m4sek** [comment]
**Status:** working
**TV Model:** 65SM8600PLA
**webOS Version:** 4.9.5-6 (goldilocks2-gympie)
**Software Version:** 05.20.06

**Comment by stephank** [comment]
Working:
```
    "hardware_id": "HE_DTV_W17H_AFADABAA",
    "product_id": "OLED55C7V-Z",
    "core_os_release": "3.9.0-62709",
    "webos_manufacturing_version": "06.00.25",
```

**Comment by mateusz-as** [comment]
Status: working
TV Model: OLED55C9PLA
webOS Version: 4.9.3-5340808 (goldilocks2-gyerongsan)
Software Version: 05.10.25

**Comment by ftc2** [comment]
Status: working

```
    "hardware_id": "HE_DTV_W17H_AFADATAA",
    "product_id": "OLED65B7P-U",
    "core_os_release": "3.9.0-62709",
    "webos_manufacturing_version": "06.00.25",
```

**Comment by arejaytee** [comment]
Status: Working
`    "hardware_id": "HE_DTV_C20P_AFADABAA",
    "product_id": "K650USW",
    "core_os_release": "5.3.0-19",
    "webos_manufacturing_version": "04.30.40"`

**Comment by iggardo** [comment]
Status: Not working
TV Model: 32LF650V-ZB
webOS Version: 2.2.3-171 (beehive-biscayne) 
Software Version: 04.06.40
![IMG_20220120_180025](https://user-images.githubusercontent.com/45078098/150392988-5d7b9527-b97a-404c-a359-97fc2ff40dca.jpg)

**Comment by DpunktKpunkt** [comment]
32LF650V has webOS 2.0, don't know if rooting ever worked with webOS < 3.0.

**Comment by odelrio** [comment]
**Status:** working
**TV Model:** OLED55G16LA
**webOS Version:** 6.2.0-32 (kisscurl-kavir)
**Software Version:** 03.21.05

**Comment by darkxdd** [comment]
Status: working
TV Model: 55UK7500PTA
webOS Version: 4.4.0-9(goldilocks-gorce)
Software Version: 5.40.09

**Comment by DpunktKpunkt** [comment]
slightly offtopic: New fw version appeared for some models (fw 5.40.26 for UK6xxx, UK7xxx, SK8xxx, SK9xxx, OLED (C8, E8, G8, W8), dated 01/21/2022).
Coincidence or does it contain fix for rootmytv?

**Comment by Informatic** [comment]
> slightly offtopic: New fw version appeared for some models (fw 5.40.26 for all webOS 4.0 models: LK5xxx, LK6xxx, UK6xxx, UK7xxx, SK8xxx, SK9xxx, OLED (C8, E8, G8, W8), dated 01/21/2022). Coincidence or does it contain a fix against rootmytv?

Should be safe.

**Comment by Sanek78** [comment]
**Status:** working
**TV Model:** 60UM7100PLB
**webOS Version:** 4.9.5-8
**Software Version:** 05.20.08

**Comment by DpunktKpunkt** [comment]
Status: working```"hardware_id": "HE_DTV_W20P_AFADABAA",
    "product_id": "65UN71006LB",
    "core_os_release": "5.3.0-2202",
    "webos_manufacturing_version": "04.30.57",```

**Comment by Meroje** [comment]
working

    "hardware_id": "HE_DTV_W16K_AFADABAA",
    "product_id": "OLED55B6V-Z",
    "core_os_release": "3.4.0-5709",
    "webos_manufacturing_version": "05.60.25",

**Comment by Wtcheesecakes** [comment]
Status: working
"hardware_id": "HE_DTV_W21U_AFADATAA",
"product_id": "OLED55B16LA",
"core_os_release": "6.2.0-31",
"webos_manufacturing_version": "03.21.00",

**Comment by FcrNah** [comment]
Status: working
    "hardware_id": "HE_DTV_W16N_AFADABAA",
    "product_id": "55UH7507-ZA",
    "core_os_release": "3.4.0-5908",
    "webos_manufacturing_version": "05.60.25",

**Comment by yourfriendoss** [comment]
**Status:** working
**TV Model:** 50UM7500PLA
**webOS Version:** 4.9.5-9
**Software Version:** 05.20.09

**Comment by Patriciooo** [comment]
Status: not working
TV Model: 49sm9000PLA
webOS Version: 4.9.5-10
Software Version: 05.20.015

Finally is all fine but i dont have the channel app. I can see the tv in http://ip:3000
Thanks

**Comment by DpunktKpunkt** [comment]
> Status: not working TV Model: 49sm9000PLA webOS Version: 4.9.5-10 Software Version: 05.20.15
This FW version was released 5 days ago. LG now presumably begins with countermeasures.

**Comment by Jarikk** [comment]
Status: working
  
"hardware_id": "HE_DTV_W20O_AFABABAA",
"product_id": "OLED65GXRLA",
"core_os_release": "5.3.0-2204",
"webos_manufacturing_version": "04.30.58",

**Comment by carvalr** [comment]
**Status:** not working
**TV Model:** 65SM8600PSA
**webOS Version:** 4.9.5-10
**Software Version:** 05.20.15

It runs but the Homebrew Channel app is not installed.

**Comment by ElDavoo** [comment]
Status: working

"hardware_id": "HE_DTV_W20P_AFADABAA",
"product_id": "50UN70006LA",
"core_os_release": "5.3.0-2204",
"webos_manufacturing_version": "04.30.58",

**Comment by Sharkiss17** [comment]
Status: Not working
TV Model: 65C7V-Z
webOS Version: 3.9.0-62709 (dreadlocks2-dudhwa)
Software Version: 06.00.25
![IMG_20220311_181105](https://user-images.githubusercontent.com/101416926/157907437-bec1042a-7475-42a2-8486-fa2f35a1745a.jpg)

**Comment by raujaku** [comment]
Status: working    
    "hardware_id": "HE_DTV_W20O_AFABATAA",
    "product_id": "OLED77CXPUA",
    "core_os_release": "5.3.0-22",
    "webos_manufacturing_version": "04.30.55",

**Comment by RobertoD91** [comment]
Status: Working
```
"hardware_id": "HE_DTV_W16M_AFADABAA",
"product_id": "OLED55E6V-Z",
"core_os_release": "3.4.0-6107",
"webos_manufacturing_version": "05.60.15",
```

PS: ssh start only after a few reboots, i deleted developer app before using rootmytv

**Comment by Jorgepfm** [comment]
**Status**: working
**TV Model**: 49UJ7500-SA
**webOS Version**: 3.9.0-62709
**Software Version**: 06.00.25

**Status**: not working
**TV Model**: 43UM7300PSA
**webOS Version**: 4.9.5-10
**Software Version**: 05.20.15

**Comment by Jammal333** [comment]
Status: Downgrade + Root Not working
TV model: OLED65E97LA
WebOS Version: 4.9.5-10 (goldilocks2-gympie)
Software Version: 05.20.15
Additional information: TV shutdown, LED blink's four times and reboot.

**Comment by jc82517** [comment]
> Status: Downgrade + Root Not working
> 
> TV model: OLED65E97LA
> 
> WebOS Version: 4.9.5-10 (goldilocks2-gympie)
> 
> Software Version: 05.20.15
> 
> Additional information: TV shutdown, LED blink's four times and reboot.

Same.

**Comment by insideone** [comment]
**Status:** working

```json
"hardware_id": "HE_DTV_W21U_AFADATAA",
"product_id": "OLED48A1RLA",
"core_os_release": "6.2.0-37",
"webos_manufacturing_version": "03.21.30",
```

**Comment by mwhdc** [comment]
Status: Working
```
    "board_type": "O18_ATSC",
    "hardware_id": "HE_DTV_W18O_AFABATAA",
    "product_id": "OLED65C8PUA",
    "core_os_release": "4.4.1-1004",
    "webos_manufacturing_version": "05.45.04",
```
Post rooting, it looks like Home Assistant can no longer send notifications (toasts) or control the TV as a media_player.

**Comment by Informatic** [comment]
> Edit: Home Assistant webostv component couldn't connect to TV after rooting, fixed by restarting HA

@mwhdc If you followed [that step](https://github.com/RootMyTV/RootMyTV.github.io#mitigation-note) you'll need to revert it. (LG Connect Apps is *the* API used by Home Assistant)

If you didn't, you may need to reregister your TV in Home Assistant. RootMyTV/Homebrew Channel itself doesn't break LG Connect Apps. (and I can personally confirm Home Assistant control *works* on rooted TVs)

**Comment by GuillaumeSmaha** [comment]
Status: Working
```json
    "hardware_id": "HE_DTV_W21O_AFABATAA",
    "product_id": "OLED65G16LA",
    "core_os_release": "6.2.0-35",
    "webos_manufacturing_version": "03.21.20",
```

**Comment by bernikr** [comment]
**Status**:  not working
**TV Model**: 55UN71003LB
webOS Version: 5.3.0-2209 (jhericurl-jasper)
Software Version: 04.30.62

**Comment by RobertoD91** [comment]
**Status:** Not working
**TV Model:** 32LQ63006LA
**webOS Version:** 7.1.0-33 (mullet-maria)
**Software Version:** n.a. - i see only ESN VSN Windevine

TV reboots after exploit, but homebrew is not installed

![IMG_6148](https://user-images.githubusercontent.com/298514/162580821-c6d82e46-dc41-4e2a-b7db-79836f68246d.jpg)

**Comment by cactode** [comment]
Status: Working
```
    "hardware_id": "HE_DTV_W21O_AFABATAA",
    "product_id": "OLED65C1AUB",
    "core_os_release": "6.2.1-2414",
    "webos_manufacturing_version": "03.25.25",
```
Also tested with OLED55C1AUB. Forgot to pull detailed info for that one, but it was fully updated to 03.25.25 and working fine.

**Comment by Bofundu** [comment]
Status: **Not working**
TV Model: 49UM7400PLB
webOS Version: 4.9.3-5390407
Software Version: 5.20.15

**Comment by matebenyovszky** [comment]
Status: V2 working nicely after removing developer mode app:
    "hardware_id": "HE_DTV_W21P_AFADATAA",
    "product_id": "70UP76703LB",
    "core_os_release": "6.2.0-40",
    "webos_manufacturing_version": "03.21.45",
Thank you very much!

**Comment by krasatos** [comment]
Status: Not Working
Exploit runs, tv reboots but no homebrew channel installed.

LG GX 65
Software Version: 04.30.61
Webos Version: 5.3.0-2206 (jhericurl-jasper)
![WhatsApp Image 2022-05-07 at 4 28 25 PM](https://user-images.githubusercontent.com/1773624/167256468-775c6000-0481-4556-9f06-2f26b1b89bbf.jpeg)

**Comment by jlvlnzl** [comment]
Status: Working
TV Model: OLED55B8SSC
webOS Version: 4.3.0-9604 (goldilocks-goonbang)
Software version: 05.30.10

**Comment by MeatyPB** [comment]
Status: Not Working
TV Model: OLED55CX5LB
WebOS Version: 5.3.0-2206 (jhericrul-jasper)
Software Version: 04.30.61

**Comment by jlvlnzl** [comment]
Status: Working
TV Model: OLED55B8SSC
webOS Version: 4.4.0-12 (goldilocks-gorce)
Software version: 05.40.26

**Comment by eduardhojbota** [comment]
Status: working
```
"hardware_id": "HE_DTV_W19P_AFADABAA",
"product_id": "65UM7510PLA",
"core_os_release": "4.9.5-9",
"webos_manufacturing_version": "05.30.09",
```

**Comment by mrtotoli** [comment]
Status: Not Working

Model: OLED55CXPSA
WebOS: 5.3.0-2206
Software: 04.30.61

**Comment by VB304** [comment]
Status: not working
TV Model: 43UP7750PSB
webOS Version: 6.3.0-9 (kisscurl-kinglake)
Software Version: 03.30.15

seems like ip:3000 is active but the channel app is not installed, I also noticed Step 1's LG Connect Apps is missing

![IMG20220627195426](https://user-images.githubusercontent.com/64369117/175937475-5e0b1e38-ccc8-4b20-b482-c1518a376e7f.jpg)
![IMG20220627195437](https://user-images.githubusercontent.com/64369117/175937498-ff01f192-de45-473c-8b4b-36e16def8240.jpg)
![image](https://user-images.githubusercontent.com/64369117/175937771-7b9f0fcb-b86e-43eb-b301-26c298f3fbeb.png)

**Comment by MaTyyyJ** [comment]
Status: Not working
Model: 55UN74003LB
WebOS: 5.4.1-9 (jhericurl-jebil)
Software: 04.40.09

IP:3000 - Hello world, but after reboot no new app installed.

**Comment by Ralf-Rollan** [comment]
Status: Not working
Model: OLED77C1PSA
Software: 03.33.11
Exploit runs, tv reboots but no homebrew channel installed.

**Comment by frankk74** [comment]
Status: Not Working
Model: 65UM7300AUE
WebOS: 4.9.6-114 (goldilocks-gympie)
Software: 05.25.35

Exploit runs, tv reboots but no homebrew channel installed.
ip:3000 working

**Comment by haakonstorm** [comment]
Status: **Not working :-(**
Model: `49UM7100PLB`
webOS: `4.9.5-14 (goldilocks-gympie)`
Software: `05.20.35`

- Exploit runs, tv reboots but no homebrew channel installed.
- Ports: (not sure if relevant)

<img width="697" alt="Screenshot of iTerm2 (19-08-2022, 21-12-00)" src="https://user-images.githubusercontent.com/193781/185690628-f05ae718-9d4b-4564-9610-4398dc1b12f5.png">

**Comment by wolfygit** [comment]
Status: Not Working
TV Model: 49SM8200PLA
webOS Version: 4.9.5-14(goldilocks2-gympie)
Software Version: 05.20.35

**Comment by m0uh85** [comment]
Status: not working / xxx:3000 reachable / no error during install
TV Model: 65NANO867NA
webOS: 5.4.1-8 (jhericurl-jebil)
Software: 04.40.08

I don't know which was the previous version but with this it was working.... I did a update (without any permit from my side) afther the update it was still installed and working but no SSH & Piccap on/off was not installable so I did a factory reset and tried to re-root it again... so now its not working :(

**Comment by c10l** [comment]
Status: not working / xxx:3000 reachable / no error during install
TV Model: OLED55CX6LA
webOS: 5.4.1-10 (jhericurl-jebil)
Software: 04.40.10

First time trying to root. Exploit seemed to run fine over website. Upon reboot, nothing happened - no notifications, no homebrew channel. I get `Hello world` on `x.x.x.x:3000` and can ping the TV but no telnet or SSH or anything else.

**Comment by YasirPie** [comment]
Status: not working
TV Model: 43UN71003LB
webOS: 5.4.1-9 (jhericurl-jebil)
Software: 04.40.09

Installed and reboot, no homebrew-app. (any updates?)

**Comment by evolution555** [comment]
Status: Not Working
TV Model: oled65cx5lb
webOS Version: 5.4.1-10(jhericurl-jebil)
Software Version: 04.40.10

**Comment by kerta1n** [comment]
Status: Not working
TV Mode: OLED65C1AUB
webOS Version: 6.3.1-411 (kisscurl-kinglake) [this does not follow what the README states]
Software Version: 03.33.11

IP address:3000 works, but no homebrew app

**Comment by bkbartk** [comment]
Status: Not Working
49UM7050PLF
by LG
Firmware: 05.20.35 
Installed, asked for permissions and reboot(tried 3 times), no homebrew-app

**Comment by hero3210** [comment]
Please update the instructions as “Software Version” & “webOS Version” are in a different place now. 

All Settings > Support > Software Update > Software Version

All Settings > General > Devices > TV Management > TV Information > webOS TV Version
6.3.1-411 (kisscurl-kinglake)


————-
Status: Not Working.
webOS TV Version: 6.3.1-411 (kisscurl-kinglake).
Software Version: 03.33.11
Model: OLED77C1PVB

**Comment by Suspect-byte** [comment]
Status: Not working/ No brew App Store working
Model: OLED55CX6LA
webOS TV Version: 5.4.1-10 (jhericurl-jebil)
Software Version: 04.40.10

Is there any fix? Thanks in advance guys great effort and great stuff. Hopefully it will work on my tv as well :)

**Comment by bkbartk** [comment]
If you just want homebrew and install some additional apps. The alternative method works really easy. 
https://www.webosbrew.org/

**Comment by YasirPie** [comment]
> If you just want homebrew and install some additional apps. The alternative method works really easy. 
> 
> https://www.webosbrew.org/

I have the same version. But I need to root it for Hyperion. Any idea what the status is. Or if there ever is gonna be an update for this one?

**Comment by ektorasdj** [comment]
Status: Not Working.
webOS TV Version: 4.9.0
Software Version: 05.20.35
Model: 43LM6300PLA

Cant downgrade message : Software update cannot be applied to this tv (the software 05.00.03 is correct for my tv)

**Comment by LifelessLife** [comment]
Works with Whiterat manual exploit HB channel and root/ssh/telnet.

webOS TV: 4.9.5-14 (goldilocks2-gympie)
modelName : 50UM7500PLA
sdkVersion : 4.9.0
firmwareVersion : 05.20.35
boardType : K5LP_DVB_EU
otaId : HE_DTV_W19P_AFADABAA

**Comment by alphadijkstra** [comment]
**Status:** Not Working
**webOS TV:** 4.9.5-14 (goldilocks2-gympie)
**modelName :** OLED55C9PLA
**firmwareVersion :** 05.20.35
**tv ip port 3000 returns hello world**: yes
**homebrew channel available**: no

**Comment by bombenbodo** [comment]
**Status:** working

   **Hardware ID:** HE_DTV_W16H_AFADABAA
    **Model:**  OLED65G6V-Z
    **WebOS version:** 3.4.0-6007 (dreadlocks-digya)
    **Firmware version:** 05.60.30

Finally. After _GetMeIn_ didn't work when trying a few years ago, I can now enjoy hombrew and get rid of all the ads.

**Comment by dannkunt** [comment]
Status: Not working
Model: 55UN68006LA
WebOS: 5.4.1-9 (jhericurl-jebil)
Software: 04.40.09

IP:3000 - Hello world, but after reboot no new app installed.

**Comment by pinewoodranger** [comment]
Status: Not working
Model: 65UK6470PLC
WebOS: 4.4.2-10 (goldilocks-gorongosa)
Software: 05.50.10

Luna call failed, denied method call download.

![20221002_001924](https://user-images.githubusercontent.com/30835477/193455595-41530a70-37a3-4897-9085-c204c59194ce.jpg)

**Comment by ledgardeneruk** [comment]
Status: Not Working/No store
TV Model: oled65cx5lb
webOS Version: 5.4.1-10(jhericurl-jebil)
Software Version: 04.40.10

**Comment by Pandalism** [comment]
**Status:** Not Working/No store
**TV Model:** OLED55CX5LB
**webOS Version:** 5.4.1-1201(jhericurl-jebil)
**Software Version:** 04.40.16

**Comment by hero3210** [comment]
> 77C1

you can downgrade to Software Version: 03.21.10 like I did & it'll work.

**Comment by soxism** [comment]
Status: Not working/ No brew App Store working
Model: OLED55CX6LA
webOS TV Version: 5.4.1-10 (jhericurl-jebil)
Software Version: 04.40.10

**Comment by GeorgeBroughton** [comment]
Status: Not Working/No store
Model: 55QNED81
webOS TV Version:  7.2.0-32 (mullet-marine)
Software Version: Not displayed in this version

It reboots the display, but other than that, there is no observable difference between before and after the exploit is run.

**Comment by MattL0** [comment]
I think people got the idea that v2 is not working anymore . That should be closed

**Comment by noderat** [comment]
**Status:** Working
**TV Model:** OLED77C1PUB
**webOS VersVer:** 6.2.1-2414
**Software VersVer:** 03.25.25

Would love to know if anybody has gotten anything newer than 03.25.25 to work on a LG C1.

**Comment by pinakkolada** [comment]
> I think people got the idea that v2 is not working anymore . That should be closed

Are you serious? Why? How comes? What can we do instead then?

**Comment by alphadijkstra** [comment]
> > I think people got the idea that v2 is not working anymore . That should be closed
> 
> Are you serious? Why? How comes? What can we do instead then?

A workaround is mentioned in this thread.
As of now your best bet for custom application installation is:
- Turn on developer mode via the LG TV
- Install an application (ipk file) through webos-dev-manager (https://github.com/webosbrew/dev-manager-desktop)
- Keep refreshing your LG developer mode token (can be automated by scripting in for instance home automation)

**Comment by pinakkolada** [comment]
> * ipk file

Thank you so much, mate. I will try the method you mentioned above. Hopefully, it will work...Cheers.

**Comment by mpeter50** [comment]
> I think people got the idea that v2 is not working anymore . That should be closed

Additionally to what @pinakkolada said, it might be useful to document additional TV models and version numbers to be able to know which works and which doesn't.
However, that would possibly work better in a spreadsheet format, instead of everyone making a new comment.

**Comment by pinakkolada** [comment]
> > > I think people got the idea that v2 is not working anymore . That should be closed
> > 
> > 
> > Are you serious? Why? How comes? What can we do instead then?
> 
> A workaround is mentioned in this thread which comes down to: As of now your best bet for custom application installation is
> 
>     * Turn on developer mode via the LG TV
> 
>     * Install an application (ipk file) through webos-dev-manager (https://github.com/webosbrew/dev-manager-desktop)
> 
>     * Keep refreshing your LG developer mode token (can be automated by scripting in for instance home automation)

It worked! Thank you so much. I hope you can help me with one more thing. I have an app from google store (apk file). Do you have any ideas how to convert this apk file to ipk file so that I can install on LG TV? Is there a way?

**Comment by DpunktKpunkt** [comment]
If the App is in WebOS brew repo, download it from https://repo.webosbrew.org/apps/

**Comment by alphadijkstra** [comment]
> Do you have any ideas how to convert this apk file to ipk file so that I can install on LG TV? Is there a way?

Sadly WebOS is not Android based. It seems it is linux based though (with my limited understanding). So as far as I know there is no easy conversion.

**Comment by pinakkolada** [comment]
> > Do you have any ideas how to convert this apk file to ipk file so that I can install on LG TV? Is there a way?
> 
> Sadly WebOS is not Android based. It seems it is linux based though (with my limited understanding). So as far as I know there is no easy conversion.

Thank you...What about any unofficial websites where we could download more apps for LG TV? Do you know any? Device manager for WebOS does not have many apps.

**Comment by mpeter50** [comment]
@pinakkolada > I have an app from google store (apk file). Do you have any ideas how to convert this apk file to ipk file so that I can install on LG TV? Is there a way?

I wouldn't think it is possible to use Android apps directly on WebOS, these work largly differently. _Maybe_ if the app is simple enough.
It would most probably be only feasible by using an Android emulator, like [Waydroid](https://wiki.archlinux.org/title/Waydroid).
Like the other user said, WebOS is Linux based, so it might be possible to run Waydroid (or an other Android emulator made for Linux) on it, but that would need some (or probably more) work to get working. It also has to be considered whether the TV has enough resources (CPU power, RAM) to run the emulator _and_ the app (you probably won't be able play fancy graphics games on it, but the limits might be reached much earlier), I have no idea how much resources an LG TV has.

> Thank you...What about any unofficial websites where we could download more apps for LG TV?

I would be interested in that too.
The Homebrew Channel app can work with certain 3rd party "app stores" (repositories), but I don't know if there are any.
However this is probably not the place for discussing this.

**Comment by MattL0** [comment]
Try the crashd method guys..

**Comment by Daycrawler** [comment]
Haven't seen mine here, so adding:
TV model: OLED55CX3LA
webOS TV Version: 5.4.1-1201
Software Version: 04.40.20
Same issue as most - script went by fine, rebooted fine, no app. Also WiFi had some connection issues after said reboot, but that's minor.

**Comment by abjugard** [comment]
```
"hardware_id": "HE_DTV_W19O_AFABABAA",
"product_id": "OLED65C9PLA",
"core_os_release": "4.9.5-14",
"webos_manufacturing_version": "05.20.35",
```

[rootmy.tv](https://rootmy.tv/) failed, jailbreaking via crashd method was successful, but the RootMyTV attempt left a script on my TV which ran after the crashd method eventually got to a certain stage, which downgraded homebrew channel to an incompatible version. Solved by upgrading homebrew channel directly via homebrew channel settings.

**Comment by filamento** [comment]
Status: WORKING!
TV Model: 32LJ610V-ZD
WebOS Version: 3.9.0-63006 (dreadlocks2-dudhwa)
Software Version: 06.00.15

I entered the "rootmy.tv" website and clicked the link above the slider. It asked for permission, I answered "YES", and then it said something about exploiting vulnerable app "facebook login" or something like that (it went very fast).

Homebrew works correctly and I've been able to install Jellyfin! YAY!

**Comment by vinismarques** [comment]
To anyone wondering what is the `crashd` method, where to find it and if it works, I just tried and it worked great! It's more technical and manual work, but if you want some adventure you can find the method in the pinned messages of rootmytv channel in the [OpenLGTV Discord](https://discord.gg/nKQW6FPWeM).

**Comment by mpeter50** [comment]
For those preferring to have some personal privacy, it is also available in the [OpenLGTV Matrix room](https://matrix.to/#/#openlgtv:netserve.live). [Direct link to message](https://matrix.to/#/!PUzIXDEvLraKgbAcpg:netserve.live/$GTmqLMcJY8rww2QhvYsHrt1jTC6NAV1Tt3cGcgmMQOo?via=hackerspace.pl&via=matrix.org&via=mozilla.org).

**Comment by Heisenberg2980** [comment]
Status: Not working
Model: 82UN85006LA
WebOS: 5.4.1-903 (jhericurl-jebil)
Software: 04.40.18

IP:3000 - Hello world, but after reboot no new app installed.

**Comment by EdgeSync** [comment]
Status: Not Working
TV Model: OLED65BX6LB
webOS Version: 5.4.1-903 (jhericurl-jebil)
Software Version: 04.40.18

Run exploit, everything looks okay, no error messages I can see, reboots, no homebrew app. Assuming exploit doesn't work on this TV.

**Comment by rayanmestiri** [comment]
Status: Not Working
TV Model: OLED48CX6LB
webOS Version: 5.4.1-15 (jhericurl-jebil)
Software Version: 04.40.20

Run exploit, accept permission, it reboots, no homebrew app.
"Hello world" on port 3000 via web browser.
Can connect via telnet on port 3000 too.

**Comment by i3olo** [comment]
Status: Not Working
TV Model: 65UP77009LB
webOs Version: 6.3.1-417 (kisscurl-kinglake)
Software Version: 03.33.25

Run exploit, accept permission, it reboots, no homebrew app.
"Hello world" on port 3000 via web browser.
Cannot connect via telnet on port 3000.
![image](https://user-images.githubusercontent.com/16158259/200829448-9b12ab80-22e8-42ba-b838-952333fb5be2.png)

**Comment by briangullo** [comment]
Status: Not Working/No store
TV Model: 86NANO75UPA
webOS Version: 6.3.1-411 (kisscurl-kinglake)
Software Version: 03.33.11

Run explain, accept permission, it reboots, no homebrew app.

**Comment by jotabf** [comment]
Status: Not Working 
TV Model: 43UQ7500PSF
webOS Version: 7.2.0-39 (mullet-marine)
Software Version: 03.21.05

It looks like it's installed, but no app appears.

**Comment by jkurei** [comment]
My TV is a 49UN73906LE, with webOS 5.3.1-29.

**Comment by M3C-Owx** [comment]
RootMyTV v2 Method
Status: Not Working, device restarts but not store and no root, cannot be downgraded
TV Model: OLED55BX9LB
webOS Version: 5.4.1-8 (jhericurl-jebil)
Software Version: 4.40.08

---------------------------
Crashd method - working!

**Comment by vdumbrav** [comment]
Status: Not working
TV Model: 43UQ75006LF
webOS Version: 7.2.0
Software Version: 03.21.20

**Comment by pannal** [comment]
Status: Working
OLED65B9SLA
05.30.15 (4.9.7-12) after previous root via V2 method.

**Comment by Bleala** [comment]
Status: Not Working
TV model: OLED65C17LB 
webOS version: 6.x.x
Firmware Version: 03.33.11

**Comment by FlaminSarge** [comment]
Status: Rootmy.tv Not Working, downloads things and reboots but does not change any behavior/Homebrew ch missing
TV model: OLED65C2AUA
webOS version: 7.2.0-44 (mullet-marine)
Firmware version: 03.21.30

**Comment by GeographicCone** [comment]
Status: **Working**
TV Model: **50<ins>UK6</ins>950PLB**
Hardware ID: **HE_DTV_W18A_AFADABAA (LM18A)**
webOS Version: **4.3.0-9703**
Software Version: **05.30.10** _(goobang)_

Exploit & Homebrew installation also fully worked on **05.40.26** _(gorce)_ but I downgraded since there was an issue with the signal source randomly switching itself to _Live TV_ within a couple of seconds from other inputs (such as HDMI) or apps. This was possibly due to the crude ad block I implemented with [dnsmasq](https://thekelleys.org.uk/dnsmasq/doc.html) interfering with the TV's attempts to report signal source changes back to the mothership, however it's only a guess as trying to pinpoint the exact cause wasn't worth the effort: it seems 05.40.26 brought exactly nothing worth upgrading to over 05.30.10.

**Comment by bogdandragoiu** [comment]
Status: Not Working (downloads, reboots, but Homebrew is missing)
TV model: OLED48A19LA
webOS version: 6.3.2
Firmware Version: 03.33.26

**Comment by sambirempo** [comment]
Status: Not Working (downloads,reboots, no app, no telnet)
TV model: OLED55B6V-Z
webOs version: 3.4.2.570909 (deadlocks-digya)
Firmware Version: 05.70.35

**Comment by coldfix** [comment]
**Status:** Not Working (downloads,reboots, no app, no telnet)
**TV model:** 43UM7450PLA
**webOs Version:** 4.9.5-10 (goldilocks2-gympie)
**Software Version:** 05.20.15

**Comment by Just-a-lurker** [comment]
Rootmy.tv web method
Status: Not Working, device restarts but no homebrew app
TV Model: 43UN7190PTA
webOS Version: 5.4.1-904 (jhericurl-jebil)
Software Version: 4.40.19

But Crashd works like a charm

**Comment by AppleMayExist** [comment]
**Status:** Not working - downloads and reboots. :3000 hello world works, but nothing else.
**TV Model:** 75UH656T-TC
**webOS Version:** 3.4.2-610708 (dreadlocks-digya)
**Software Version:** 05.70.30

**Comment by tim-rex** [comment]
OLED65C8PLA
WebOS 4.4.2-10 (goldilocks-goringosa)
Firmware: 05.50.10

Rootmy.tv appears to install correctly. No observable errors. Reboot occurs without issue. No app visible.

**Comment by kazigk** [comment]
Working!

```
    "hardware_id": "HE_DTV_W21A_AFADATAA",
    "product_id": "55NANO756PA",
    "core_os_release": "6.0.1-4553",
    "webos_manufacturing_version": "03.11.35",
```

**Comment by jannekayra** [comment]
OLED77CX6LA
webOS TV Version 5.4.2-25 (jhericurl-jebil) 
Software version 04.40.70

Does not work! No error but no app after reboot.

**Comment by wohmiguel** [comment]
65NANO75UPA
webOS TV Version: 6.3.2-423 (kisscurl-kinglake)
Software Version: 03.34.55

- Does not work, it finishes and reboots but the app just does not show up; I'm using another method to wait but hopefully this gets fixed
- Hello world at http://localhost:3000 on the TV works
- No notifications on startup too

**Comment by leonardohn** [comment]
**Status**: Not working
**TV Model**: 43UK6510PSF
**WebOS Version**: 4.4.2-10 (goldilocks-gorongosa)
**Software Version**: 05.50.10

**Comment by istinet** [comment]
43NANO793NE
webOS TV version: 5.4.2-25 (jhericurl-jebil)
Software version: 04.40.70
Not working. 
After installation and reboot, it does not show notification, app not installed.

**Comment by brietman** [comment]
> Status: Not working TV Model: 32LF650V-ZB webOS Version: 2.2.3-171 (beehive-biscayne) Software Version: 04.06.40 ![IMG_20220120_180025](https://user-images.githubusercontent.com/45078098/150392988-5d7b9527-b97a-404c-a359-97fc2ff40dca.jpg)
I have the same issue. Running WebOS 04.06.75 on a LG 32LF630V. "Service does not exist". Maybe because we have no Facebook app? I will try the GetMeIn method.

**Comment by Tractorxd** [comment]
Status: not working
TV Model: 55UK6500MLA
webOS Version: 4.4.2-10 (goldilocks-gorongosa)
Software Version: 05.50.10

Will look into GetMeIn, I dont know what that is but maybe that will work?

**Comment by tobmir24** [comment]
Status: **not working**
TV Model: 55SK8500LLA
webOS Version: 4.4.2-10 (goldilocks-gorongosa)
Software Version: 05.50.10

![grafik](https://github.com/RootMyTV/RootMyTV.github.io/assets/134965642/c2707d06-b204-4638-b0a1-61bffe00a377)

**Comment by working-name** [comment]
Status: **not working**
TV Model: OLED55B8PUA
WebOS Version: 4.4.2-10 (goldilocks-gorongosa)
Software version: 05.50.10

`errorText: Denied method call` for both exploits tried.

**Comment by wolfygit** [comment]
"hardware_id": "HE_DTV_W20P_AFADABAA",
    "product_id": "75NANO796NF",
    "core_os_release": "5.4.3-36",
    "webos_manufacturing_version": "04.41.45",

**Comment by h4lm3th34d** [comment]
Status: Not Working - Denied Method Call \"Download"\
TV Model: 55SK8500PTA
webOS Version: 4.4.2-11 (goldilocks-gorongosa)
Software Version: 05.50.15

**Comment by yecarrillo** [comment]
**Status:** Root status retained after upgrade from 04.30.40
**TV Model:** OLED55CXPDA
**webOS Version:** 5.5.0-1102 (jhericurl-jervisbay)
**Software Version:** 04.50.52

Accidentally upgraded by my daughter from version [04.30.40] (https://github.com/RootMyTV/RootMyTV.github.io/issues/18#issuecomment-1044556453) (where TV was originally rooted) to [04.50.52]. 

```    
    "board_type": "O20_DVB", 
    "hardware_id": "HE_DTV_W20O_AFABABAA",  
    "product_id": "OLED55CXPDA", 
    "core_os_release": "5.5.0-1102", 
    "webos_manufacturing_version": "04.50.52",

**Comment by romerotsvk** [comment]
> Status: **not working** TV Model: 55SK8500LLA webOS Version: 4.4.2-10 (goldilocks-gorongosa) Software Version: 05.50.10
> 
> ![grafik](https://private-user-images.githubusercontent.com/134965642/241790324-c2707d06-b204-4638-b0a1-61bffe00a377.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ5NzMxMzIsIm5iZiI6MTcwNDk3MjgzMiwicGF0aCI6Ii8xMzQ5NjU2NDIvMjQxNzkwMzI0LWMyNzA3ZDA2LWIyMDQtNDYzOC1iMGExLTYxYmZmZTAwYTM3Ny5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTExJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDExMVQxMTMzNTJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03Y2UzMWYyMDUxYjkwZmE1YjBiZTBlYzY1YzM3ZjA4M2FjNzQ1NmMzODk4YjJmMDhlM2MwYzY1OGNiNTZiNDcyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.WPvClCVIk3HyzwNQMwdhb2TmPuGTBoZNGwXxLo-BEc8)

I have the same problem - is there a solution?

**Comment by throwaway96** [comment]
@yecarrillo 

You are saying that you retained root after updating to that version, not that you successfully ran RootMyTV on that version, right? I think your post is likely to confuse people.

@bartjackbakker

RootMyTV v2 will work on W20O 04.30.40 but not 04.50.52.

@romerotsvk 

If you have webOS 4+ and haven't updated in at least a couple months, [crashd](https://gist.github.com/throwaway96/e811b0f7cc2a705a5a476a8dfa45e09f) will work. If you have webOS <4, currently the only reliable method is [NVM modification](https://gist.github.com/throwaway96/827ff726981cc2cbc46a22a2ad7337a1). (GetMeIn *may* work on webOS <3.5, but you should **not** use the binary on XDA. The hardcoded commands it runs are not safe.)

**Comment by fonsito** [comment]
> **Status**: Working **TV Model**: OLED65C15LA **webOS Version**: 6.2.0-27 (kisscurl-kavir) **Software Version**: 3.20.80

Not working on 


    "hardware_id": "HE_DTV_W21O_AFABATAA",
    "product_id": "OLED65C15LA",
    "core_os_release": "6.4.0-1804",
    "webos_manufacturing_version": "03.40.87"

**Comment by idevsoftware** [comment]
**Status:** Working
**TV Model:** OLED55B9PSB

Hi, I'm not sure this is the right place to post this but I want to report success on rooting my OLED55B9 running webOS 05.40.45 (as of this writing this is the latest available version released a few weeks ago) with `dejavuln-autoroot`:

```
    "hardware_id": "HE_DTV_W19H_AFADATAA",
    "product_id": "OLED55B9PSB",
    "core_os_release": "4.10.0-17",
    "webos_manufacturing_version": "05.40.45",
```

I am leaving this here as corresponding the model info in [CanI.RootMy.TV](https://cani.rootmy.tv/?q=OLED55B9PSB) shows 05.40.20 as the latest known rootable version for my model.

**Comment by ObiKeahloa** [comment]
**Status**: Not Working
**TV Model**: OLED55C8PTA
**webOS Version**: 4.3.0-9806 (goldilocks-goobang)
**Software Version**: 05.30.20

**Comment by PRIHLOP** [comment]
**Status**: not working(unfortunately) on **55UF8507-ZB**
 "device_name": "lm15u",
    "hardware_id": "HE_DTV_W15U_AFADABAA",
    "hardware_revision": "DV_1ST",
    "product_id": "55UF8507-ZB",
    "core_os_kernel_version": "3.10.23-p.21.biscayne.lm15u.11",
    "core_os_name": "Rockhopper",
    "core_os_release": "2.2.3-2155",
    "core_os_release_codename": "beehive-biscayne",
    "encryption_key_type": "prodkey",
    "webos_api_version": "4.1.0",
    "webos_build_id": "2155",
    "webos_imagename": "starfish-dvb-secured",
    "webos_manufacturing_version": "04.06.25",
    "webos_name": "webOS TV",
    "webos_prerelease": "",
    "webos_release": "2.2.3",
    "webos_release_codename": "beehive-biscayne"

**Comment by yecarrillo** [comment]
> [@yecarrillo](https://github.com/yecarrillo) did you root it on the 5.5.0 version or 4.30.40?

Rooted on webos_manufacturing_version: 04.30.40 as reported on firts post: https://github.com/RootMyTV/RootMyTV.github.io/issues/18#issuecomment-1044556453

> [@yecarrillo](https://github.com/yecarrillo)
> 
> You are saying that you retained root after updating to that version not that you successfully ran RootMyTV on that version, right?

Yes. Root status retained after native upgrade by answering 'Yes' to 'Do you want to upgrade?' on the UI. Second post edited for better comprehension
https://github.com/RootMyTV/RootMyTV.github.io/issues/18#issuecomment-1849011531
